gradle/compile-js-multiplatform.gradle

```diff
@@ -5,15 +5,14 @@
 apply from: rootProject.file('gradle/node-js.gradle')
 
 kotlin {
-    targets {
-        fromPreset(presets.js, 'js') {
-            // Enable built-in test runner only for IR target.
-            // These runners don't support changing js module name change.
-            if (js.hasProperty("irTarget") && irTarget != null) {
-                irTarget.nodejs()
-                irTarget.compilations['main']?.dependencies {
-                    api "org.jetbrains.kotlinx:atomicfu-js:$atomicfu_version"
-                }
+    js {
+        if (it.hasProperty("moduleName")) {
+            moduleName = project.name
+        }
+        if (it.hasProperty("irTarget") && it.irTarget != null) {
```

Comment:
when writing non-trivial construct please drop a comment why it is needed

Reply:
Done

Reply:
Done